fox_news_scraper/load_csv_fn.py

```diff
@@ -1,8 +1,12 @@
+# pylint: disable=unused-import
```

Comment:
Good idea to include this so pylint doesn't get mad at us for the grequests import